src/Domain/MigrationStep/s150_ProductVariationMigration/Entity/FamilyVariant.php

```diff
@@ -37,14 +37,14 @@ class FamilyVariant
     private $familyCode;
 
     public function __construct(
-        ?int $id,
+        ?int $id = null,
```

Comment:
Not optional if the next parameter is mandatory.

Reply:
Did you remember the persist method in FamilyVariantRepository? :wink: